en/peripherals/gripper.md

```diff
@@ -0,0 +1,106 @@
+# Gripper Peripherals
+
+![High-load gripper example](../../assets/hardware/grippers/highload_gripper_example.jpg)
+
+Grippers are a commonly used mechanism for UAV delivery applications. There are multiple types of interfaces (e.g. PWM, CAN) for different products in the market.
+
+## Supported Gripper Types
+
+PX4 supports the following Gripper types, check the individual page to learn how to integrate them.
+
+- [Servo Gripper with PWM](gripper_servo.md)
+
+:::note
+The [Roboclaw driver](../modules/modules_driver.md#roboclaw) isn't supported as a Gripper yet. However in the future it may get supported as a Gripper.
+:::
+
+## Gripper Actuator Mapping
+
+To enable the output of the Gripper, set the Function of the output port the gripper is connected to as `Gripper`.
+
+:::note
+Make sure to set the correct PWM frequency (usually commercial servo / grippers only support 50Hz)! If you mis-configure the frequency you risk damaging the gripper.
+:::
+
+![Gripper output mapping](../../assets/config/gripper/qgc_gripper_output_setup.png)
+
+## Gripper Action Joystick Button Mapping in QGC
+
+QGC provides the Button mapping interface, which allows to map delivery mechanism actions to specific buttons of the [Joystick](../config/joystick.md) connected to QGC.
```

Comment:
Can an RC controller switch be mapped in http://docs.px4.io/main/en/config/flight_mode.html#flight-mode-selection

I'm guessing not, but we should make that clear somewhere.